### PR TITLE
Update documentation & publish github->action

### LINKED ISSUE #63: Planning next LPVS release (1.0.2)
Let's start to plan the next LPVS feature set.
Internally we discussed that it could be good if we provide the possibility for developers to use LPVS as a **GitHub Action step**.
We are investigating such a possibility, but first of all, I would like to ask your opinion about the idea.

### PR DESCRIPTION
**Pull request by tdrozdovsky**
# Description

Update README.md and publish action.

Fixes #63 

## Type of change

- [x] Documentation update
- [x] CI system update

# How Has This Been Tested?
**Test Configuration**:
* Java: v11
* LPVS Release: v1.0.2

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by o-kopysov — APPROVED** [review]
LGTM

**Review by MoonkiHong — APPROVED** [review]
+1

File: README.md [hunk 1]
```diff
@@ -14,164 +14,203 @@ OpenSource code [refers](https://en.wikipedia.org/wiki/Open-source_software) to
...
+docker-compose up
+```
+
+Stop _LPVS_ services by `docker-compose` (before Compose V2)
+```bash
+docker-compose down -v --rmi local
+```  
+Stop _LPVS_ services by `docker compose` (after Compose V2)
+```bash
+docker-compose down
```
Comment (tiokim) [inline README.md:103]:
```suggestion
docker compose down
```

File: docker-compose.yml [hunk 2]
```diff
@@ -3,8 +3,7 @@ version: '3.4'
 services:
 
   lpvs:
-    build:
-      context: .
```
Comment (tiokim) [inline docker-compose.yml:7]: I think we can leave these for `docker compose build` command.

**Review by tiokim — COMMENTED** [review]
PTAL

File: docker-compose.yml (hunk 2 above)
Comment (tdrozdovsky) [reply docker-compose.yml:7]: I'll check

File: README.md (hunk 1 above)
Comment (tdrozdovsky) [reply README.md]: Thank you! Done!

File: README.md [hunk 3]
```diff
@@ -14,164 +14,203 @@ OpenSource code [refers](https://en.wikipedia.org/wiki/Open-source_software) to
...
+ - ./mysql-lpvs-data:/var/lib/mysql # db storage by default it is directory in root of repository with name mysql-lpvs-data
+```
+#### 2.2 Run `lpvs` and `mysqldb` docker images by docker-compose
+Start _LPVS_ services by `docker-compose` (before Compose V2)
+```bash
+docker-compose up -d
+```
+Start _LPVS_ services by `docker compose` (after Compose V2)
+```bash
+docker-compose up
```
[the remark this answers is not in the document]
Comment (tdrozdovsky) [reply README.md]: Thank you! Done!

File: README.md [hunk 4]
```diff
@@ -14,164 +14,203 @@ OpenSource code [refers](https://en.wikipedia.org/wiki/Open-source_software) to
...
+
+---
+
+## Quick start
+
+### 1. Setting up your project to interact with _LPVS_ 
+
+_LPVS_ license scan shall be enabled on your project via GitHub Webhooks:
+
+1. [Creating a personal access token (`github.token`)](https://docs.github.com/en/enterprise-server@3.4/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-personal-access-token)
```
[the remark this answers is not in the document]
Comment (tdrozdovsky) [reply README.md]: Thank you! Done!

File: Dockerfile [hunk 5]
```diff
@@ -33,4 +33,4 @@ EXPOSE 7896
 WORKDIR /$PRJ_DIR/target
 
 # Run application in container
-CMD ["java", "-jar", "lpvs-1.0.2.jar"]
+CMD ["java", "-jar", "lpvs-*.jar"]
```
Comment (tiokim) [inline Dockerfile:36]: @tdrozdovsky My lpvs container can't find the `lpvs-*.jar` file. Does your container work accordingly?

**Review by tiokim — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 5 above)
Comment (tiokim) [reply Dockerfile:36]: Please see #166

**Review by tiokim — APPROVED** [review]
LGTM! Thanks for the contribution for v1.1!

**Pull request merged**